config/initializers/assets.rb

```diff
@@ -1,5 +1,13 @@
 # Be sure to restart your server when you modify this file.
 
+Rails.application.config.assets.configure do |env|
```

Comment:
Is this only needed for development/test?  If so, wrap this in a `unless Rails.env.production?`

Alternatively: add `next unless ENV.key?("SPROCKETS_CACHE")` after this line to set this only when `SPROCKETS_CACHE` is present in the environment.